.gitignore

```diff
@@ -14,6 +14,7 @@
 .vscode
 !/gsap-svelte/.vscode
 !/quasar/.vscode
+!/static/.vscode
```

Comment:
Nit: let's remove this line since we don't intend to commit `static/.vscode/settings.json`, and so that we don't end up committing that accidentally.